dashboard/src/containers/AppNewContainer/AppNewContainer.tsx

```diff
@@ -31,7 +33,9 @@ function mapStateToProps(
   };
 }
 
-function mapDispatchToProps(dispatch: Dispatch<IStoreState>) {
+function mapDispatchToProps(
+  dispatch: ThunkDispatch<IStoreState, null, ChartsAction & ServiceCatalogAction>,
```

Comment:
what does it mean `ChartsAction & ServiceCatalogAction` here?

Reply:
Ok so I was wrong here.

My intention was to say that the dispatch inside the mapStateToProps was going to receive either a chart or a serviceCatalog action, but reality is that it was receiving an async action in all those cases.

Also, I was using an intersection when I actually meant union, but it was working because of the reason described above, async (ThunkActions) are already supported in the type so it does not matter what I put as last parameter that it did not change anything.

I have noticed that actually setting a specific value i.e `ServiceCatalogAction` as last type parameter is not that valuable in the mapStateToProps since it can be confusing and also, once you define the signature of the actionCreator this value can be overridden (see https://github.com/kubeapps/kubeapps/pull/650/files#diff-2fc417299c3e2ab5151030d339ab8dfcR107) so I changed it to use the global `Action` instead. 

Then, once we adapt our action creators we can use a proper type https://github.com/kubeapps/kubeapps/issues/658